src/lib.rs

```diff
@@ -284,9 +323,36 @@ impl Wizer {
     /// rather than per-instance.
     ///
     /// Defaults to `false`.
-    pub fn allow_wasi(&mut self, allow: bool) -> &mut Self {
+    pub fn allow_wasi(&mut self, allow: bool) -> anyhow::Result<&mut Self> {
+        // Check if custom linker support is already enabled, if so return error
```

Comment:
Personally, I don't think this comment is really saying anything that the code itself doesn't already, so I think we can remove it.